src/content/meta/glossary.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
it is a meta page so it will be in -> https://docs.giantswarm.io/meta/ (and reference from different doc entries, but not in the sidebar menu) If we don't change it